web/src/containers/ActivityDetailCostAllocateFFP.test.js

```diff
@@ -52,8 +52,8 @@ describe('the activities cost allocation FFP component', () => {
           statePersonnel: [
             {
               years: {
-                '1066': { amt: 100 },
-                '1067': { amt: 1000 }
```

Comment:
Tweak the test data to exercise the allocation math.  Lots of update required as a result of this, but higher confidence that the math is right.